packages/components/src/Components/NotAuthorized/NotAuthorized.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon403 from './icon-403';
+import {
+    Title,
+    Button
+} from '@patternfly/react-core';
+import { withRouter } from 'react-router-dom';
```

Comment:
I think that you could do this without props with `useHistory` (https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/hooks.md#usehistory) - you'd need to update react-router-dom to 5.x, I don't know if that's OK though.

Reply:
Is react-router-dom needed at all here actually? You can call `history.back()` directly from the browser without importing anything